src/CodeGeneration/CodeGeneration.LowLevelClient/ApiEndpoints/cat.snapshots.json

```diff
@@ -8,6 +8,7 @@
       "parts": {
         "repository": {
           "type" : "list",
+          "required" : true,
```

Comment:
have to send a PR in the main repos for this: https://github.com/elastic/elasticsearch/issues/17216